### DISCUSSION TITLE
Html to docx with inline styles

### DISCUSSION THREAD

**Discussion by KaioSpa** [jgm/pandoc#9367] [Q&A]
Hey, I couldn't find how to do this, so here I am...

I have a rich text editor where my users can input text, and it's saved as HTML data in my database. Users have the option to change aspects of the text, like this:
```
<p style="text-align: right;"><span style="font-size: 8pt;">Sample text</span></p>
```

I also want to give them the option to save this data as a DOCX file. However, when I make the conversion to DOCX, it does not keep this formatting in the final DOCX file. Is there a command to keep the formatting?

I'm creating a temporary file with the HTML and converting it using this process:
```
$process = new Process([
    'pandoc',
    '--from',
    'html',
    '--to',
    'docx',
    '--mathml',
    $inputFilePath,
    '-o',
    $outputFilePath
    ]);

    $process->run();
```

**Comment by jgm** [comment]
Answer at the issue you also created (#9368).
